Projekt/src/app/app.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from '@angular/common/http';
+import { IDeck } from "./deck.model";
+import {HttpHeaders} from '@angular/common/http';
```

Comment:
`HttpHeaders` and `HttpClient` are imported from the same package, so you can merge these lines into one.